### PR TITLE
fix: custom button component to fix inconsistency

### PR DESCRIPTION
**Pull request by vatsalsinghkv**
## Related Issue

Closes: #644 

#### Describe the changes you've made ✍

- Have created a custom `Button` component, with the following props:
    - `variant`: `solid` (default) | `outline`
    - `to`: null (default) - pass the `to` prop with url to use this button as a `Link`
    - `size`: null (default) | `sm` - used in the navbar
    - `...rest`: rest button props can be passed
- Replaced the buttons with the `Button` component
- Commented some unused code to avoid linting errors

**This is is just for previewing the button styles**

### CODE REVIEW

File: src/components/SingleClubEvent.jsx [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
   const nav = useNavigate();
 
   const cardButtonHandler = () => {
     nav(`/clubs/${club._id}`);
   };
-  const expand = () => {
-    document.getElementById(`less${club._id}`).classList.add("hidden");
-    document.getElementById(`more${club._id}`).classList.remove("hidden");
-  };
+  // const expand = () => {
```
Comment (IAmTamal) [inline src/components/SingleClubEvent.jsx:18]: You can remove these commented codes as they were previously been used but not now at all.

**Review by IAmTamal — CHANGES_REQUESTED** [review]
The rest of the changes are looking good to me however I do have some minor changes for you. Please look at the video below (turn up the volume too)

https://github.com/IAmTamal/Milan/assets/72851613/fea46162-502e-436e-9035-3c73c11333eb

**Comment by vatsalsinghkv** [comment]
@IAmTamal about the size of the button on the banner (hero section), do I have to reduce the size of buttons everywhere or just on the banner?

**Comment by IAmTamal** [comment]
> @IAmTamal about the size of the buttons on the banner (hero section), do I have to reduce the size of buttons everywhere or just on the banner?

Just on the home screen. You can compare it to the currently deployed [site](https://milaan.vercel.app/)

**Review by IAmTamal — APPROVED** [review]
**Amazing work,**
This looks good to me, will be merged soon 🎉.

**Pull request merged**